src/components/TextClassification/index.js

```diff
@@ -1,7 +1,8 @@
-import React, { useState } from "react"
+import React, { useState, useEffect, useRef } from "react"
 import getTaskDescription from "../../utils/get-task-description.js"
 import SampleContainer from "../SampleContainer"
 import NLPAnnotator from "react-nlp-annotate/components/NLPAnnotator"
+import * as datasetHelper from "../../utils/dataset-helper"
```

Comment:
So for new code the best pattern is to import the function directly from a file, when I refactored I didn't have time to break up this file, but the best pattern would be to do something like:

```javascript
import getTextFromSample from "../../utils/get-text-from-sample"
```

The ["Helper" pattern](https://en.wikipedia.org/wiki/Helper_class) should be avoided in all new code ([see more in this stack overflow post](https://softwareengineering.stackexchange.com/questions/247267/what-is-a-helper-is-it-a-design-pattern-is-it-an-algorithm)).

Reply:
Do i break it in a folder ?